R/models.R

```diff
@@ -0,0 +1,76 @@
+cloudml_model_exists <- function(gcloud, name) {
+
+  arguments <- (MLArgumentsBuilder(gcloud)
+                ("models")
+                ("list"))
+
+  output <- gcloud_exec(args = arguments())
+  pasted <- paste(output$stdout, collapse = "\n")
+
+  suppressWarnings({
+    output <- readr::read_table2(pasted)
```

Comment:
I removed the use of readr in favor `utils::read.table` (to avoid the heavyweight dependency which we'll pay for compilation of on first deploy). Here's the commit: https://github.com/rstudio/cloudml/commit/a56a27274f28bc962771b039e9e1e23cce2ae283

Let's do that here as well